spring-integration-core/src/main/java/org/springframework/integration/channel/AbstractMessageChannel.java

```diff
@@ -116,6 +119,16 @@ public boolean isStatsEnabled() {
 		return this.statsEnabled;
 	}
 
+	protected AbstractMessageChannelMetrics getMetrics() {
+		return this.channelMetrics;
+	}
+
+	@Override
+	public void configureMetrics(ConfigurableMetrics metrics) {
```

Comment:
Aren't you going to make it configurable per `<channel>`? I mean that it is available for JavaConfig, but isn't for XML